packages/babel-plugin-proposal-object-rest-spread/src/index.js

```diff
@@ -356,6 +356,10 @@ export default function() {
           props = [];
         }
 
+        if (t.isSpreadElement(path.node.properties[0])) {
```

Comment:
@nicolo-ribaudo Thanks for the suggestion! Seems that I eventually figured it out :)